app/controllers/concerns/devise_parameter.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module DeviseParameter
+  protected
+
+  def update_allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:firstname, :lastname, :email, :password) }
```

Comment:
To match with the database column, it should be like 
```suggestion
    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:first_name, :last_name, :email, :password) }
```